sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ConnectionPolicy.java

```diff
@@ -29,7 +29,7 @@
     private RetryOptions retryOptions;
     private boolean enableEndpointDiscovery = true;
     private List<String> preferredLocations;
-    private boolean usingMultipleWriteLocations;
+    private boolean usingMultipleWriteLocations = true;
```

Comment:
It should not be ported to V3. Please track it.

Reply:
Ok